catalog-info.yaml

```
@@ -0,0 +1,18 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'edx-django-utils'
+  description: "EdX utilities for Django Application development"
```

[nit] I think we're supposed to use lowercase initial letter for edX, even at the beginning of a sentence. I don't care very much, though. :-)
```suggestion
  description: "edX utilities for Django Application development"
```